module/VuFind/src/VuFind/Log/LoggerAwareTrait.php

```diff
@@ -29,22 +29,45 @@
 
 namespace VuFind\Log;
 
+use Psr\Log\LoggerInterface;
+use Psr\Log\LogLevel;
+
 use function get_class;
 
 /**
- * Extension of \Laminas\Log\LoggerAwareTrait with some convenience methods.
+ * Extension of PSR-3 LoggerAwareTrait with some convenience methods.
```

Comment:
Updating this to match:

```suggestion
 * Implementation of PSR-3 \Psr\Log\LoggerAwareTrait with some additional convenience methods.
```